utils/api_server.py

```diff
@@ -80,7 +80,23 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
     text_outputs = []
     for output in final_output.outputs:
         text_outputs.append(output.text)
-    ret = {"text": text_outputs}
+
+    try:
+        logprobs = [output.logprobs for output in final_output.outputs]
+    except Exception as e:
+        logprobs = []
+        print(e)
+        print("Could not extract logprobs")
+
+    logprobs = logprobs[0]
```

Comment:
just checking why do we only pick the first entry here?

Reply:
That's just because of the way vllm returns the logprobs! 

`logprobs = [output.logprobs for output in final_output.outputs]` is a list of lists.

Reply:
Hmm I'll go test this more thoroughly! Tested this in the [defog-llm-server](https://github.com/defog-ai/defog-llm-server/blob/main/llm/api_server.py) repo in prod - will try to figure out what's the issue here.

Thanks for flagging!

Reply:
@wongjingping: I ran this on the H100 prod server and seems to work okay!